rfcs/20200612-stream-executor-c-api.md

```diff
@@ -88,40 +88,62 @@ plan to wrap only a subset of key StreamExecutorInterface functionality.
 See proposed C API below:
 
 ```cpp
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 typedef SE_Stream_st* SE_Stream;
 typedef SE_Event_st* SE_Event;
 typedef SE_Timer_st* SE_Timer;
 typedef TF_Status* (*TF_StatusCallbackFn)(void*);
 
+#ifndef TF_BOOL_DEFINED
+#define TF_BOOL int8_t
```

Comment:
I just realized `c_api.h` in TensorFlow is using `unsigned char` for `bool`. We should probably do the same. Not a big change, but if you are fixing the other comment I left on your commit, you can update this one as well.
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/c/c_api.h#L42

Reply:
Done. Thanks for the thorough review Anna!